src/set_theory/surreal/basic.lean

```diff
@@ -261,6 +271,9 @@ instance : has_neg surreal  :=
   (λ x ox, ⟦⟨-x, ox.neg⟩⟧)
   (λ _ _ _ _ a, quotient.sound (pgame.neg_congr a))⟩
 
+noncomputable def _root_.ordinal.to_surreal (o : ordinal) : surreal :=
+mk _ (numeric_to_pgame o)
```

Comment:
The linter will want a docstring on this.

Also, it should be easy to show that this definition is monotonic.

Reply:
Alright, there goes.